Gateway/Request/SepaDirectDebitDataBuilder.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Buckaroo\Magento2\Gateway\Request;
+
+class SepaDirectDebitDataBuilder extends AbstractDataBuilder
+{
+    public function build(array $buildSubject): array
+    {
+        parent::initialize($buildSubject);
+
+        $paymentInfo = $this->getPayment();
+
+        $customerBic = $paymentInfo->getAdditionalInformation('customer_bic');
+        $customerIban = $paymentInfo->getAdditionalInformation('customer_iban');
+        $customerAccountName = $paymentInfo->getAdditionalInformation('customer_account_name');
+
+        $data = [
+            'iban' => $customerIban,
```

Comment:
Maybe we should add collectdate / mandateDate parameters

Reply:
Thanks, in develop branch we don't add it, but I will check if the old version send it somehow.